### PR TITLE
Add logic for testprocess

### PR DESCRIPTION
**Pull request by KhaninPasha0488**

### CODE REVIEW

File: src/screens/Testing/TestProcess/index.tsx [hunk 1]
```diff
@@ -1,44 +1,91 @@
...
+import {AllTestOptionsMoc} from '@src/Mocs/Testing';
+import {AppButton} from '@src/components/ui/AppButton';
 import {ViewFlexCenter, CountQuestionBox, ViewBlock} from './styles';
-import {ProgressBar} from '../../../components/ProgressBar/index';
+import {ProgressBar} from '@src/components/ProgressBar';
+import {ProgressType} from '@src/components/ProgressBar/ProgressView';
 
 export const TestProcess = () => {
+  const [numAnswer, setNumAnswer] = useState<number>(1);
+  const CurrentTest = AllTestOptionsMoc.filter(e => e.id === numAnswer);
```
Comment (antoniosk10) [inline src/screens/Testing/TestProcess/index.tsx:18]: currentTest

File: src/Mocs/Testing.ts [hunk 2]
```diff
@@ -28,3 +30,54 @@ export const TestOptionsMoc = [
   'Java Script',
   'React',
 ];
+
+export const AllTestOptionsMoc: IAllTestOptionsMoc = [
```
Comment (antoniosk10) [inline src/Mocs/Testing.ts:34]: allTestOptionsMoc

File: src/screens/Testing/TestProcess/index.tsx [hunk 3]
```diff
@@ -1,44 +1,91 @@
...
+    if (numAnswer < AllTestOptionsMoc.length) {
+      setNumAnswer(numAnswer + 1);
+    }
+  };
+  const onPressSkipAnswer = () => {
+    if (numAnswer < AllTestOptionsMoc.length) {
+      setNumAnswer(numAnswer + 1);
+    }
+  };
+  const ProgressData: ProgressType[] = [...Array(AllTestOptionsMoc.length)].map(
```
Comment (antoniosk10) [inline src/screens/Testing/TestProcess/index.tsx:32]: progressData

File: src/screens/Testing/TestProcess/index.tsx [hunk 4]
```diff
@@ -1,44 +1,91 @@
...
+        },
+  );
+  const timeSeconds = (CurrentTest[0].timer % 60).toString();
+  const timeMinutes = Math.floor(CurrentTest[0].timer / 60).toString();
+  const strCorrect = CurrentTest[0].content.answers.correct.split('//');
+  const strWrong = CurrentTest[0].content.answers.wrong.split('//');
+  const arrAnswers = strCorrect.concat(strWrong);
+  const randomAnswers = arrAnswers.sort(() => Math.random() - 0.5);
+  const answerType = CurrentTest[0].type;
+  const Question = CurrentTest[0].content.question;
```
Comment (antoniosk10) [inline src/screens/Testing/TestProcess/index.tsx:56]: question

File: src/screens/Testing/TestProcess/index.tsx [hunk 5]
```diff
@@ -1,44 +1,91 @@
...
+  const TitleQuestions = CurrentTest[0].title;
```
Comment (antoniosk10) [inline src/screens/Testing/TestProcess/index.tsx:57]: titleQuestions

File: src/Mocs/Testing.ts [hunk 6]
```diff
@@ -28,3 +30,54 @@ export const TestOptionsMoc = [
...
+
+export const allTestOptionsMoc: IAllTestOptionsMoc = [
+  {
+    id: 1,
+    timer: 200,
+    title: 'NodeJS question #13',
+    content: {
+      question: 'Which core module in Node can you use for testing?',
+      answers: {
+        wrong: 'Type Script//Java Script//React',
```
Comment (antoniosk10) [inline src/Mocs/Testing.ts:42]: here will be just string

**Review by HuziMuzi — DISMISSED** [review]

File: src/screens/Testing/TestProcess/index.tsx [hunk 7]
```diff
@@ -1,44 +1,90 @@
...
+  const onPressNextAnswer = () => {
+    if (numAnswer < quizIdMocState.questions.length) {
+      setNumAnswer(numAnswer + 1);
+    }
+  };
+  const onPressSkipAnswer = () => {
+    if (numAnswer < quizIdMocState.questions.length) {
+      setNumAnswer(numAnswer + 1);
+    }
+  };
```
Comment (antoniosk10) [inline src/screens/Testing/TestProcess/index.tsx:32]: What is the difference between onPressNextAnswer and onPressSkipAnswer?

Comment (KhaninPasha0488) [reply src/screens/Testing/TestProcess/index.tsx:32]: the logic will be added to onPressSkipAnswer. Haven't decided yet.

**Review by Julia-Kovalchuk — APPROVED** [review]

**Review by HuziMuzi — APPROVED** [review]

**Pull request merged**